docs/chain_documentation/workflow/osm/data_preparation.md

```diff
@@ -1,6 +1,68 @@
 # OSM  - Extraction and transformation to GIS layers
 
+Figure 1 shows the sequence of actions that are performed to retrieve OSM data and transform them into GIS layers.
+
+![osm_extraction](../../../resources/images/chain_documentation/osm_extract_gis_layers.png)
+
+Figure 1. OSM data flow transformation
+
+The sequence starts either by specifying a place name or by specifying an OSM bbox.
+
+If the user set a place name, the [Nominatim API](https://nominatim.org) is requested  to search in the OSM database all
+objects that match the given text.
+A geojson file is returned with a geometry representation of the OSM objects.
+The first Polygon or MutiPolygon found in the geojson is collected and its bounding box (BBox) is computed.
+
+The BBox is used to create an Overpass query.
+
+```xml
+[maxsize:1073741824][bbox:47.6377566,-2.7976946,47.6530896,-2.7694425];
+(
+	node;
+	way;
+	relation;
+);
+(._;>;);
+out;
+```
+
+The response of the query is stored in an OSM xml file. 
+The XML file is loaded in a relational spatial database (H2GIS or PostGIS).
+Thanks to the [H2GIS OSM driver](http://www.h2gis.org/docs/1.5.0/OSMRead/) that performs this task. 
```

Comment:
No point after `task`

Reply:
don't understand

Reply:
The sentence is not finished, so no point after the word "task"

Reply:
Thanks to the [H2GIS OSM driver](http://www.h2gis.org/docs/1.5.0/OSMRead/) that performs this task, 10 tables are produced.